build_image.sh

```diff
@@ -90,7 +117,9 @@ fi
 popd
 
 # get new Managed Image from the packer output
-managed_image_id="$(grep -Po '(?<=ManagedImageId\: )[^$]*' ${packer_log})"
+managed_image_id="$(grep -Po '(?<=ManagedImageId\: )[^$]*' logs/${packer_log})"
+export OS_IMAGE_RESOURCE_ID=$managed_image_id
+env > /tmp/build_image_env.env
```

Comment:
probably want to remove this test line 

might be interesting to dump a ./test_image_env.sh for use with the test_cc_image.sh that ONLY contains:

#!/bin/bash

export OS_IMAGE_RESOURCE_ID=$managed_image_id